docs/source/markdown/podman-image-scp.1.md

```diff
@@ -0,0 +1,125 @@
+% podman-image-scp(1)
+
+## NAME
+podman-image-scp - Securely copy an image from one host to another
+
+## SYNOPSIS
+**podman image scp** [*options*] *name*[:*tag*]
+
+## DESCRIPTION
+**podman image scp** saves an image to either **docker-archive**, **oci-archive**, **oci-dir** (directory with oci manifest type), or **docker-dir** (directory with v2s2 manifest type) on the local machine, and then copies the saved image to a remote host specified either by **--remote** or **--connection**
+default is **docker-archive**. **podman image scp** writes to STDOUT by default and can be redirected to a
+file using the **output** flag. The **quiet** flag suppresses the output when set.
+**podman image scp** utilizes **podman save** and **podman load**.
+Note: `:` is a restricted character and cannot be part of the file name.
+
+**podman [GLOBAL OPTIONS]**
+
+**podman imaege scp [GLOBAL OPTIONS]**
+
+**podman image scp [OPTIONS] NAME[:TAG]**
+
+## OPTIONS
+
+#### **--remote**, **-r**
```

Comment:
These options are confusing to me?  --remote is a global option and should not be documented here?